src/gate/parserGnet/parserGnet.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * \brief Implements translation from Liberty format file to GNet.
+ * \author <a href="mailto:anushakov@edu.hse.ru">Aleksander Ushakov</a>
+ */
+
+#pragma once
+
+#include "gate/model/gnet.h"
+
+#include <kernel/yosys.h>
+
+#include <map>
+#include <vector>
+
+struct NetData {
+    std::vector<eda::gate::model::GNet> combNets;
+    std::vector<eda::gate::model::GNet> memNets;
+};
+
+void translateModuleToGNet(
+    const std::pair<Yosys::RTLIL::IdString, Yosys::RTLIL::Module*> &m,
+    NetData &net);
+
+void translateDesignToGNet(
+    const Yosys::RTLIL::Design &des,
+    NetData &vec);
+
+void translateLibertyToDesign(
+    const char* namefile,
+    NetData &vec);
+
+std::map<eda::gate::model::GNet*, std::vector<uint64_t>> truthTab(
+    NetData &vec);
```

Comment:
There is no point to do all at once in the API.
```suggestion
std::vector<uint64_t> buildTruthTable(const GNet &net);
```